### ISSUE TITLE
Need to set small learning rate in deformal conv layer? 

### ISSUE THREAD

**Issue by xyl576807077** [chengdazhi/Deformable-Convolution-V2-PyTorch#9]

**Comment by chengdazhi** [comment]
For the conv layer that outputs the offset, setting its learning rate to 1/10 of other layers does not influence the final result on the coco dataset. However, on other datasets like ImageNet VID, we have found that doing this makes training stabler.

**Comment by lzrobots** [comment]
So does that mean for coco we can set all the layers same learning rate?

**Comment by chengdazhi** [comment]
yes, should yield same performance
